app/index.html

```diff
@@ -234,12 +208,11 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
        <div class="row add-bottom">
           <div class="twelve columns">
             <h2 class="title-margin-bottom">Media</h2>
-               <h2>Archives</h2>
-               <p>We look forward to our Archives section, coming soon!  You'll be able to browse photos of many generations of Cornell Fencing.<br>
-               Follow us on social media!
-               </p>
-
-               <h2>Social Media</h2>
+              <h3>Email Newsletter Archives</h3>
+              
+              <p>Cornell Fencing sends out regular email updates about how the program is doing. You can read through the e-mails that have been sent and subscribe to future ones by clicking <a href="http://us13.campaign-archive.com/home/?u=6d228c291b3baeafb9ec6c712&id=812a3b5c0a">this link</a>.</p>

```

Comment:
Click [here]()-style links tend to be worse for usability and accessibility. It's better to make the link be named for the thing that it's linking to. In this case, we could do:

> You can [read through the emails that have been sent and sign up for future ones]()

Or:

> You can read through the emails that have been sent and sign up for future ones on [our mailing list]().